cli/azd/test/functional/aspire_test.go

```diff
@@ -68,7 +68,7 @@ func restoreDotnetWorkload(t *testing.T) {
 
 // Test_CLI_Aspire_DetectGen tests the detection and generation of an Aspire project.
 func Test_CLI_Aspire_DetectGen(t *testing.T) {
-	restoreDotnetWorkload(t)
+	//restoreDotnetWorkload(t)
```

Comment:
?

Reply:
unwanted... I usually need to do this to re-create test snapshots...

Reply:
You should get on my sudo plan, it's actually really nice!  I just need to set the `AZD_TEST_DOTNET_WORKLOAD_USE_SUDO` and it is nice.

Reply:
I did, but it didn't help me for using the UI `run test` button.  Did you figure a way to make that one to work?

![image](https://github.com/Azure/azure-dev/assets/24213737/5c4016f8-5838-4d58-b9f3-5856d0dd375e)


The `USE_SUDO` hack worked for invoking the test from the console :)...